package.json

```diff
@@ -18,7 +18,7 @@
     "jshint": "^2.9.5",
     "mocha": "^5.0.4",
     "mock-require": "^3.0.1",
-    "nyc": "^11.4.1"
+    "nyc": "^11.6.0"
```

Comment:
Since this is only a development dependency (i.e. it is not a transitive dependency), it is quite safe to update it for the release.